src/modules/roles/domain/services/safe-role-deletion.domain.service.ts

```
@@ -76,10 +83,35 @@ export class SafeRoleDeletionDomainService {
+  /**
+   * Delete all permissions associated with the role
+   */
+  private async deleteRolePermissions(roleId: string): Promise<void> {
+    this.logger.log(`Deleting permissions for role ${roleId}`);
+
+    try {
+      await Promise.all([
+        this.permissionVmRepository.deleteByRoleId(roleId),
+        this.permissionServerRepository.deleteByRoleId(roleId),
+      ]);
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)